stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -56,7 +56,7 @@ use crate::util_lib::boot::boot_code_id;
 fn advance_to_nakamoto(
     peer: &mut TestPeer,
     test_signers: &TestSigners,
-    test_stackers: Vec<TestStacker>,
+    test_stackers: &[TestStacker],
```

Comment:
Somewhere in this file, can you add a basic unit test (or extend the existing ones) so they load and validate the reward set from the chainstate?  Just so we have some targeted test coverage on these new methods, in addition to the indirect ones we get from querying it via RPC? Thanks!

Reply:
We get them both from the RPC, but also all of the existing nakamoto coordinator tests -- the nakamoto coordinator always uses the same path as the RPC endpoint.

Reply:
Just trying to understand what you're asking for here-- the nakamoto coordinator loads its reward sets from the Chainstate DB whenever it uses the `OnChainProvider`, which is true in all of the non-unit coordinator tests. So this is covered in several different integration tests. The functionality of actually loading the reward set state in the coordinator is inherently an integration. In 2.x, its an integration between the coordinator, pox-3, the on chain provider, and the SortitionDB. In Nakamoto, its an integration between the coordinator, pox-4, the on chain provider, the SortitionDB, and the block processing methods. So, this functionality at least has to be tested via integration tests (which it is). The functionality of computing the reward set is just a unit (it's just a function applied to the PoX-4 state), and that _is_ tested via unit tests (implemented in the PR that this one built off of).